### ISSUE TITLE
TypeError: Cannot read property '0' of undefined

### ISSUE THREAD

**Issue by koo5** [digitalbazaar/jsonld.js#294]
when trying to parse
```
<file://bn614X>  <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil>.
```

full testcase: 
```js
var done = 0;
(function wait () {if (!done) setTimeout(wait, 1000);})();

const jsonld = require('jsonld');

const line = "<file://bn614X>  <http://www.w3.org/1999/02/22-rdf-syntax-ns#first> <http://www.w3.org/1999/02/22-rdf-syntax-ns#nil>."
jsonld.fromRDF(line, {format: 'application/n-quads'}, (err, doc) => {
            if (err) {
              console.log("err" + err)
            } else {
              console.log("doc"+doc)
            }
            done = true
});

```

**Comment by davidlehn** [comment]
That's a bug in the list handling code.  https://github.com/digitalbazaar/jsonld.js/pull/295 will fix this and make the expanded output should look like:
```json
[
  {
    "@id": "file://bn614X",
    "http://www.w3.org/1999/02/22-rdf-syntax-ns#first": [
      {
        "@list": []
      }
    ]
  }
]
```
I have no idea if that's correct in this case.  I'll add a test to the test suite and get some feedback.

**Comment by davidlehn** [comment]
Fixed in 1.5.4.  Thanks!
